samples/Hello/Hello.ble

```diff
@@ -5,13 +5,13 @@
 
 */
 
-// * reference and null tests
```

Comment:
All the changes to comments are because I wanted to make the capitalization consistent